integration-tests/utils/Common.ts

```diff
@@ -15,9 +15,9 @@ export class Common {
   }
 
   static openApplicationURL(applicationName: string) {
-    Common.openURL(
-      `${Cypress.env('HAC_BASE_URL')}/applications?name=${applicationName.replace('.', '-')}`,
-    );
+    if(!cy.contains('h1', applicationName, { timeout: 90000 })) {
```

Comment:
AFAIK `cy.contains` doesn't actually produce a falsey result when nothing is found, it would just throw an error after the timeout runs out

A safer solution would be to check if the current url points to the appropriate application
 `.../applications?name=${applicationName}`